packages/termux-licenses/LICENSES/X11.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (C) 1996 X Consortium
```

Comment:
Since the file has an copyright header, which varies between packages, we won't be able to add as a generic license to termux-licenses